Source/WebCore/platform/text/TextFlags.cpp

```diff
@@ -272,7 +272,12 @@ FeaturesMap computeFeatureSettingsFromVariants(const FontVariantSettings& varian
                 if (!name)
                     return { };
 
-                return tags.get(*name);
+                auto found = tags.find(*name);
+                if (found == tags.end())
+                    return { };
+
+                return Span { found->value };
+                
```

Comment:
It's also surprising that changing `const RefPtr<FontFeatureValues> fontFeatureValues` to `RefPtr<FontFeatureValues> fontFeatureValues` was needed to fix this. Since the header signature was non-const, it shouldn't have compiled, right?

Reply:
I think `get` returns the PeekType, which is not a pointer to the contained Vector but a copy of the Vector, which is a temporary and the Span is build on top of this temporary vector.

Reply:
(for the `const` in the header prototype, it's a noop : https://stackoverflow.com/questions/38660537/why-is-const-unnecessary-in-function-declarations-in-header-files-for-parameters )